pkgs/development/libraries/qt-5/5.5/qtbase/setup-hook.sh

```diff
@@ -1,135 +1,3 @@
-if [[ -z "$QMAKE" ]]; then
-
-_qtLinkDependencyDir() {
-    @lndir@/bin/lndir -silent "$1/$2" "$qtOut/$2"
-    if [[ -n "$NIX_QT_SUBMODULE" ]]; then
-        find "$1/$2" -printf "$2/%P\n" >> "$out/nix-support/qt-inputs"
-    fi
-}
-
-_qtLinkModule() {
-    if [ -d "$1/mkspecs" ]; then
-        # $1 is a Qt module
-        _qtLinkDependencyDir "$1" mkspecs
-
-        for dir in bin include lib share; do
-            if [ -d "$1/$dir" ]; then
-                _qtLinkDependencyDir "$1" "$dir"
-            fi
-        done
-    fi
-}
-
-_qtRmModules() {
-    cat "$out/nix-support/qt-inputs" | while read file; do
-      if [[ -h "$out/$file" ]]; then
-        rm "$out/$file"
-      fi
-    done
-
-    cat "$out/nix-support/qt-inputs" | while read file; do
-      if [[ -d "$out/$file" ]]; then
-        rmdir --ignore-fail-on-non-empty -p "$out/$file"
-      fi
-    done
-
-    rm "$out/nix-support/qt-inputs"
-}
-
-addToSearchPathOnceWithCustomDelimiter() {
-    local delim="$1"
-    local search="$2"
-    local target="$3"
-    local dirs
-    local exported
-    IFS="$delim" read -a dirs <<< "${!search}"
-    for dir in ${dirs[@]}; do
-        if [ "z$dir" == "z$target" ]; then exported=1; fi
-    done
-    if [ -z $exported ]; then
-        eval "export ${search}=\"${!search}${!search:+$delim}$target\""
-    fi
-}
-
-addToSearchPathOnce() {
```

Comment:
No problem -- which ones do you need? All of them? I can look by myself if you point me at where to start looking.